test/parallel/test-fs-null-bytes.js

```diff
@@ -1,21 +1,20 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
 
 function check(async, sync) {
-  var expected = /Path must be a string without null bytes/;
-  var argsSync = Array.prototype.slice.call(arguments, 2);
-  var argsAsync = argsSync.concat(function(er) {
+  const expected = /Path must be a string without null bytes/;
+  const argsSync = Array.prototype.slice.call(arguments, 2);
+  const argsAsync = argsSync.concat((er) => {
     assert(er && er.message.match(expected));
-    assert.equal(er.code, 'ENOENT');
+    assert.strictEqual(er.code, 'ENOENT');
   });
 
   if (sync)
-    assert.throws(function() {
-      console.error(sync.name, argsSync);
+    assert.throws(common.mustCall(() => {
```

Comment:
Nit: I don't think that wrap this with `common.mustCall()` is useless as it would only be wrapped when sync is truthy.